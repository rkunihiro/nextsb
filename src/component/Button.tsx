import React from "react";

import styles from "./Button.module.scss";

export const Button: React.FC = ({ children }) => {
    return (
        <>
            <div className={styles.button}>
                <span data-testid="buttonLabel">{children}</span>
            </div>
        </>
    );
};
