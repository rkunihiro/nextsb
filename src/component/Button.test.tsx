import React from "react";
import TestRenderer from "react-test-renderer";

import { Button } from "./Button";

describe("Button", () => {
    it("label", () => {
        const r = TestRenderer.create(<Button>Hello</Button>);
        expect(r.toJSON()).toMatchSnapshot();
    });
});
