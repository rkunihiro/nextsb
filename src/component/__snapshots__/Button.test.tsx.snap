// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button label 1`] = `
<div
  className="button"
>
  <span
    data-testid="buttonLabel"
  >
    Hello
  </span>
</div>
`;
